src/seedu/addressbook/data/Tagging.java

```diff
@@ -0,0 +1,56 @@
+package seedu.addressbook.data;
+
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.tag.Tag;
+
+import java.util.HashMap;
+
+public class Tagging {
```

Comment:
Lacking a header comment here. An example could be : Represents an adding or removing a Tag to/from a Person